First release for Python Wave Serial Encoder (#1)
stm32duino/Python-Wave-Serial-Encoder: README.md

# Python Wave Serial Encoder

## Manual
1. Use the dropdown menu to select the port and then click on *Connect*.

> 2. Choose the path where you want to save the file by using the *Save* button.

Reviewer: ```suggestion
Choose the path where the file will be saved by using the *Save* button.
```

> 2. Choose the path where the file will be saved the file by using the *Save* button.

Reviewer: ```suggestion
2. Choose the path where the file will be saved by using the *Save* button.
```

3. Use *Start* and *Stop* button to respectively start and stop the recording.

**Please note: to record more than one file, change the path else the old file will be overridden.**

> In case the dropdown menu doesn't detect the port, use the button *Refresh* to update the Serial Port.

Reviewer: ```suggestion
In case the dropdown menu does not detect the port, use the button *Refresh* to update the Serial Port.
```

# Python Wave Serial Encoder

## Manual
1. Use the dropdown menu to select the port and then click on *Connect*.

> 2. Chose the path where you want to save the file by using the *Save* button.

Reviewer: ```suggestion
2. Choose the path where the file will be saved by using the *Save* button.
```

> 3. Use start and stop button to respectively start and stop the recording.

Reviewer: ```suggestion
3. Use *Start* and *Stop* button to respectively start and stop the recording.
```

> Please note:  to record more then on file you have to change the path or the old file will be override.

Reviewer: ```suggestion
**Please note: to record more than one file, change the path else the old file will be overridden.**
```

> In case the dorpdown menu doesn't detect the port, use the button *Refresh* to update the Serial Port

Reviewer: ```suggestion
In case the dropdown menu doesn't detect the port, use the button *Refresh* to update the Serial Port.
```

## Installation

> The script has some dependency from some package from python. To install execute the following command:

Reviewer: ```suggestion
The script has some python packages dependencies. Install them by running:
```
